Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorAbstractEffect.h

```diff
@@ -420,6 +431,9 @@ public slots:
   QString m_Name;
```

Comment:
Clarify the description of this property:

/// Name of the effect. This name is used by various modules for getting access to an effect. This string is not displayed on the user interface and not translated.